charts/hono/README.md

```diff
@@ -414,8 +414,8 @@ helm install --dependency-update -n hono --set useCommandRouter=false eclipse-ho
 
 #### Example Implementation
```

Comment:
The above command `helm install --dependency-update -n hono --set useCommandRouter=false eclipse-hono eclipse-iot/hono` needs to be adapted so that `--set deviceRegistryExample.type=file` is in there.
Otherwise the command will fail:
```
Error: execution error at (hono/templates/hono-adapter-mqtt/hono-adapter-mqtt-vertx-secret.yaml:41:10): .Values.deviceRegistryExample.type MUST be 'file' and .Values.deviceConnectionService.enabled set to true if example Device Registry is disabled and no other Device Connection service is configured
```

Reply:
Right. That is basically what the following paragraph explains.